app/views/users/show.html.erb

```diff
@@ -1 +1,35 @@
-<h1>Display a user by id</h1>
\ No newline at end of file
+<%= stylesheet_link_tag "user_show.css" %>
+<h1>Display a user by id</h1>
+
+<div class='user'>
+        <div class="face-container">
+          <img 
+          class="face" 
+          src=<%= @user.Photo %>
+          alt="face" />
+        </div>
+        <div class="details">
+            <h1><%= @user.Name %></h1>
+            <h3>Number of posts : <%= @user.PostsCounter %></h3>
+        </div>
+    </div>
```

Comment:
- [**Optional**] You are repeating this piece of code in most of your views pages. Let's try to keep the codebase clean and DRY by using [Rails partials](https://guides.rubyonrails.org/layouts_and_rendering.html).